project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_forms.scss

```diff
@@ -92,3 +92,67 @@
 .panel-title {
   text-transform: uppercase;
 }
+
+// ========================================================================== //
+// Event Form
+// ========================================================================== //
+.node-form {
+
+  .control-label {
+    clear: both;
+    display: block;
+  }
+
+  .form-control {
+    color: $color-gray-dark;
+    font-style: italic;
+    background-color: #EBEAE4;
+    border: 0;
+    border-radius: 20px;
```

Comment:
@soniCaH 
Other issues were talking about the same thing, hence we decided to handle this here as a generic solution.